Dockerfile

```diff
@@ -12,10 +12,17 @@ RUN DEBIAN_FRONTEND=noninteractive apt-get update -q \
 
 ENV HOME /home/$USER
 
-WORKDIR $HOME/source
+WORKDIR $HOME
 
-COPY osmaxx-py $HOME/source
+# Little trick to avoid rebuilding this image over and over again.
```

Comment:
I would like to see a comment why it's better to run pip first. I know, our comments should not explain how docker works. But in reality it helps to avoid bad refactorings.

http://www.codecandies.com/2011/04/13/please-increment-the-following-counter/